zebra-consensus/src/transaction/tests.rs

```diff
@@ -569,7 +569,7 @@ fn v4_transaction_with_conflicting_sprout_nullifier_inside_joinsplit_is_rejected
         };
 
         // Sign the transaction
-        let sighash = transaction.sighash(network_upgrade, HashType::ALL, None);
+        let sighash = transaction.sighash(network_upgrade, HashType::ALL, &Vec::new(), None);
```

Comment:
Suggestion: remove the `Vec`

```suggestion
        let sighash = transaction.sighash(network_upgrade, HashType::ALL, &[], None);
```

Reply:
Done in b36577315 / #3446